src/python/pants/option/options.py

```diff
@@ -116,26 +116,26 @@ def complete_scopes(cls, scope_infos: Iterable[ScopeInfo]) -> FrozenOrderedSet[S
     @classmethod
     def create(
         cls,
-        env: Mapping[str, str],
-        config: Config,
-        known_scope_infos: Iterable[ScopeInfo],
+        *,
         args: Sequence[str],
-        bootstrap_option_values: OptionValueContainer | None = None,
+        env: Mapping[str, str],
+        config_sources: Sequence[ConfigSource] | None,
+        known_scope_infos: Sequence[ScopeInfo],
+        extra_specs: Sequence[str] = tuple(),
```

Comment:
Instead of the Options class knowing about bootstrap options and finding the extra specs there (via the `--spec-files` option), we now pass those in from the OptionsBootstrapper, so only it needs to know about bootstrap options.